website/thaliawebsite/settings.py

```diff
@@ -472,7 +472,6 @@ def from_env(
                 "django.contrib.messages.context_processors.messages",
                 "announcements.context_processors.announcements",
                 "thaliawebsite.context_processors.thumbnail_sizes",
-                "thaliawebsite.context_processors.aprilfools",
```

Comment:
I think it would also be fine to keep this (but also fine to remove it, so I don't care too much actually)

Reply:
Lets also keep this, it makes it less confusing to eventually use again